esqueleto.cabal

```
@@ -1,7 +1,7 @@
 cabal-version: 1.12
 
 name:           esqueleto
-version:        3.5.2.2
+version:        3.5.2.3
```

Adding instances is a minor bump

```suggestion
version:        3.5.3.0
```